src/js/modules/util/generateCommentList.js

```diff
@@ -0,0 +1,13 @@
+import fetchData from './fetchData.js';
+
+const renderCommentList = (id) => {
+  const url = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/yf6dqoUrsU3EfHXvC1i4/comments?item_id=${id}`;
+  const ulElement = document.querySelector('.artworks__modal-comments-list');
+
+  fetchData(url).then((data) => {
+    const renderedList = data.map((comment) => `<li class="artworks__modal-comments-item">${comment.creation_date} ${comment.username}: ${comment.comment}</li>`).join('');
+    ulElement.innerHTML = renderedList;
+  });
+};
```

Comment:
You need to handle situations where the data is null because there is no comment.  As of now, it is throwing an error for such a situation. See the image below!

![image](https://user-images.githubusercontent.com/22054464/225354378-98510ad3-6214-4820-8f39-05562f04029a.png)